src/main/java/lotto/domain/Rank.java

```diff
@@ -25,10 +28,15 @@ public int getPrizeMoney() {
         return prizeMoney;
     }
 
-    public static Rank value(int matchingCount) {
-        if (matchingCount < 3) {
-            return FIFTH;
+    public static Rank value(int matchingCount, boolean matchBonus) {
+        if (matchingCount == 5 && matchBonus) {
+            return SECOND;
         }
+
+        if (matchingCount == 5) {
+            return THIRD;
+        }
```

Comment:
두 if 문을 한개로 줄이면 2등과 2등 보너스 볼 등수를 구분할 수 있을 것 같아요 :)
```suggestion
        if (matchingCount == 5 && !matchBonus) {
            return THIRD;
        }
```